HerPublicWebsite.BusinessLogic/ExternalServices/OsPlaces/IOsPlacesApi.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace HerPublicWebsite.BusinessLogic.ExternalServices.OsPlaces
+{
+    public interface IOsPlacesApi
+    {
+        public Task GetLocalAuthorityFromPostcode(string postcode); // TODO: Create real EPC methods
```

Comment:
I don't think we need this method? We'll get the LA code from the result.
We will need something like GetAddresses(string propertyNameOrNumber, string postcode) though.